packages/react-reconciler/src/ReactFiberUnwindWork.js

```diff
@@ -216,6 +220,12 @@ export default function<C, CX>(
   }
 
   function unwindInterruptedWork(interruptedWork: Fiber) {
+    if (enableProfileModeMetrics) {
```

Comment:
Same as previous comment. Can we move this into the switch case for Profilers?

Reply:
👍